include/uv.h

```diff
@@ -244,6 +244,15 @@ typedef enum {
 UV_EXTERN unsigned int uv_version(void);
 UV_EXTERN const char* uv_version_string(void);
 
+typedef void* (*uv_malloc_func)(size_t size);
+typedef void (*uv_free_func)(void* ptr);
+
+/*
+ * If used, uv_replace_allocator must be called before any other uv 
```

Comment:
This is already mentioned in the docs and doesn't really add much here, please remove it,